docs/guides/cloud/integrations/slack.mdx

```diff
@@ -1,228 +1,143 @@
 
-   <DocsImage
-     src="/img/guides/cloud/integrations/slack/enable-slack-integration.jpg"
-     alt="Click enable under Slack Integration"
-     width={1000}
-   />
+### Per-Organization vs per-project configuration
 
-5. You'll see a popup window that requests permission for Cypress to access the
-   workspace and allows you to choose your Slack workspace and channel to
-   associate with the installation. Once you've selected a channel and allowed
-   access, the installation is complete! Cypress Cloud will post run results for
-   all projects in your organization to the specified Slack channel.
+The following describes where to go to configure Slack for an organization and a project. The remaining sections will describe the supported configuration options.
 
-For more information about how the Cypress integration for Slack collects and
-manages your information, please visit our
-[Privacy Policy](https://on.cypress.io/privacy-policy).
+#### Organization settings
 
-## Per-organization configuration
+To share Slack configuration for an organization:
 
-### Add additional Slack channels
+1. Navigate to the Integrations page for the organization you wish to configure.
+2. Within the Slack integration, click Configure.
+3. Configure the app.
 
-You can have Cypress Cloud post run results to an additional channels. To add a
-channel:
+#### Project settings
 
-1. Navigate to the **Integrations** page for the organization with the installed
-   integration.
-2. Within the Slack integration, click **Configure**.
-3. Click **Add Slack Channel**.
-4. You'll see a popup window that allows you to choose the channel to associate
-   with the organization. Cypress Cloud will post run results for all projects
-   in your organization to the new Slack channel.
+To share Slack configuration for a project:
 
-### Set notification preferences
+1. Navigate to the Project Settings page for the project you wish to configure.
+2. Click on the Integrations tab within the Project Settings page.
+3. Configure the app with the Slack Integration section.
 
-By default, Cypress Cloud will post a Slack message to each configured channel
-only for failing runs. If you'd like to change these preferences:
+### Slack channels
 
-1. Navigate to the **Integrations** page for the organization with the installed
-   integration.
-2. Within the Slack integration, click **Configure**.
-3. Under **Notifications**, select your preference for each Slack channel:
+You can have Cypress Cloud post run results to specific channels. To add a
+channel:
 
-- **Failed runs**: will notify on runs with a failed status.
-- **Passed runs**: will notify on runs with a passed status.
-- **Flaky tests**: will notify on runs that have an identified flaky test.
+1. Navigate to the Slack configuration page
+2. Click **Add Slack Channel**.
+3. You'll see a popup window that allows you to choose the channel or individual associate
```

Comment:
'individual associate', not sure what you mean here. You mean you can post to private messages?